### ISSUE TITLE
Handling 'system reserved names' like actions_step_debug

### ISSUE THREAD

**Issue by djthornton1212** [rhysd/actionlint#130]
### Situation:
Hello,
     I have a step in my workflow_call that makes use of GitHub Actions system secrets like `${{ secrets.ACTIONS_STEP_DEBUG }}`. Action lint sees this as an error reporting the following error:

> property "actions_step_debug" is not defined in object type

I understand that this error is reported because it's not referenced in the inputs section under `workflow_call:`. 
Being that this is a "system reserved" secret/variable we can't include it in the inputs without receiving the following error:

> secret name `ACTIONS_STEP_DEBUG` within `workflow_call` can not be used since it would collide with system reserved name

### How to reproduce:
Create a workflow_call and in a step use a system reserved name/secret like `${{ secrets.ACTIONS_STEP_DEBUG }}`.

### Expected behavior:
Actionlint should ignore the fact that system reserved names are included in a workflow_call as they cannot be added as inputs.

### Version Used:
Super-linter: 4.9.1
Actionlint: 1.6.10

**Comment by rhysd** [comment]
I could not reproduce this.

I prepared the following tiny workflow:

```yaml
on: push

jobs:
  some_job:
    runs-on: ubuntu-latest
    steps:
      - run: echo ${{ secrets.ACTIONS_STEP_DEBUG }}
```

and I ran:

```
actionlint ./test.yaml
```

but it did not report any error.

Is there any other pre-condition?

**Comment by rhysd** [comment]
OK, I think I understood what you meant.

> Create a workflow_call

This means a workflow like below, am I correct?

```yaml
on:
  workflow_call:

jobs:
  test:
    runs-on: ubuntu-latest
    steps:
      - run: echo ${{ secrets.ACTIONS_STEP_DEBUG }}
```

Then I could get the following error.

```
test.yaml:8:23: property "actions_step_debug" is not defined in object type {github_token: string} [expression]
  |
8 |       - run: echo ${{ secrets.ACTIONS_STEP_DEBUG }}
  |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~
```

Currently `secrets.GITHUB_TOKEN` is treated as secrets set by system. So it does not cause an error when it is used without being defined in the workflow call definitiion. I think I can add more presets.

**Comment by rhysd** [comment]
I don't know what secrets are actually reserved. If you have any official document for the list of such secrets, please let me know the pointer.

**Comment by djthornton1212** [comment]
I'll see if I can find a document
